src/sentry/integrations/slack/message_builder/image_block_builder.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+import sentry_sdk
+
+from sentry import features
+from sentry.api import client
+from sentry.charts import backend as charts
+from sentry.charts.types import ChartType
+from sentry.integrations.slack.message_builder import SlackBlock
+from sentry.integrations.slack.message_builder.base.block import BlockSlackMessageBuilder
+from sentry.integrations.slack.message_builder.time_utils import (
+    get_approx_start_time,
+    get_relative_time,
+)
+from sentry.issues.grouptype import PerformanceP95EndpointRegressionGroupType
+from sentry.models.apikey import ApiKey
+from sentry.models.group import Group
+from sentry.snuba.referrer import Referrer
+from sentry.utils.performance_issues.detectors.utils import escape_transaction
+
+logger = logging.getLogger("sentry.integrations.slack")
+
+
+class ImageBlockBuilder(BlockSlackMessageBuilder):
```

Comment:
will there be a followup PR to initialize this in the endpoint regression logic? we make the image block here but still need to send it with the normal blocks payload?

Reply:
No, it was a rebase issue. Added back the line to issues.py